README.md

```diff
@@ -1,22 +1,24 @@
-# @stripe/react-stripe
-
-[![build status](https://img.shields.io/travis/stripe/react-stripe/master.svg?style=flat-square)](https://travis-ci.org/stripe/react-stripe)
-[![npm version](https://img.shields.io/npm/v/@stripe/react-stripe.svg?style=flat-square)](https://www.npmjs.com/package/@stripe/react-stripe)
-
-> React components for Stripe.js and Stripe Elements
+## React Stripe
 
 React Stripe is a thin React wrapper around
 [Stripe Elements](https://stripe.com/docs/elements). It allows you to add
 Elements to any React app.
 
-## Docs
+[![build status](https://img.shields.io/travis/stripe/react-stripe/master.svg?style=flat-square)](https://travis-ci.org/stripe/react-stripe)
+[![npm version](https://img.shields.io/npm/v/@stripe/react-stripe.svg?style=flat-square)](https://www.npmjs.com/package/@stripe/react-stripe)
+
+# Docs
 
-TODO
+### [React Stripe Docs](https://stripe.com/docs/stripe-js/react-stripe) | [Stripe.js Reference](https://stripe.com/docs/js)
 
-Looking for the legacy `react-stripe-elements` docs? You can find those
-[here](https://github.com/stripe/react-stripe-elements/).
+- Migrating from `react-stripe-elements`? Check out our
+  [migration guide](docs/migrating.md).
+- Looking for something a little more lightweight? Check out our
```

Comment:
This feels awkward to me, because I don't connect "examples" and "lightweight" in my head. Don't have a better suggestion at the moment though. Would it be horrible to link all the different documentation in a simple list?

```
- React Stripe documentation
- React Stripe examples
- Stripe.js reference
- Legacy `react-stripe-elements` documentation
- `react-stripe-elements` migration guide
```